common/lib/common-utils/src/hashFileNode.ts

```diff
@@ -7,10 +7,21 @@
 import sha1 from "sha.js/sha1";
 import { IsoBuffer } from "./bufferNode";
 
+/**
+ * Set a hashing function to be called in place of hashFile's internal
+ * implementation when running under insecure contexts.  Not needed
+ * when running under Node.
+ * @param hashFn - The function that should be used in place of hashFile
+ */
+export function setInsecureContextHashFn(hashFn: (f: IsoBuffer) => Promise<string>) {
```

Comment:
If it isn't needed in node, why do we need to edit this file? Just to make sure both modules have the same API?

Reply:
yup that's why.  keeping it the same makes it easier to reason about the two differing platforms imo